xrpc/src/main/java/com/nordstrom/xrpc/server/ExceptionHandler.java

```
@@ -0,0 +1,9 @@
+package com.nordstrom.xrpc.server;
+
+import io.netty.handler.codec.http.HttpResponse;
+
+/** A Handler for mapping exceptions to Http Responses. */
+@FunctionalInterface
+public interface ExceptionHandler {
+  HttpResponse handle(XrpcRequest request, Exception exception) throws Exception;
```

It's not clear under what conditions this should throw an exception. Document? Ideally, this should _never_ throw exceptions (it's meant as a fallback).